experiments/integrated/fluxnet/US-MOz/US-MOz_simulation.jl

```diff
@@ -20,7 +20,7 @@ h_leaf = FT(9.5) # m
 t0 = Float64(120 * 3600 * 24)# start mid year to avoid snow
 
 # Time step size:
-dt = Float64(120)
+dt = Float64(450)
 
 # Number of timesteps between saving output:
-n = 15
+n = 4
```

Comment:
why decreased?

Reply:
we save every n*dt=3600, but since dt increased, n decreases. it would be clearer to just save every 3600 seconds and not have it depend on dt!